engine/src/main/java/org/terasology/rendering/dag/nodes/PrePostCompositeNode.java

```diff
@@ -65,13 +66,8 @@ public void initialise() {
     @Override
     public void process() {
         PerformanceMonitor.startActivity("rendering/prePostComposite");
-        prePostComposite.enable();
         sceneReflectiveRefractive = displayResolutionDependentFBOs.get(REFLECTIVE_REFRACTIVE);
 
-        // TODO: verify if there should be bound textures here.
-        WRITE_ONLY_GBUFFER.bind();
-
-        setViewportToSizeOf(WRITE_ONLY_GBUFFER);
         glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT); // TODO: verify this is necessary
 
         renderFullscreenQuad();
```

Comment:
I think the bindDisplay() and setViewportToSizeOf() lines can be safely eliminated. Just make sure to test this specific change on its own! =)